### PR TITLE
[wix-ui-icons-common] - add wix-ui-tpa icons

### PR DESCRIPTION
**Pull request by bamai**
https://jira.wixpress.com/browse/DSM-1647

### CODE REVIEW

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-icons-common**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-icons-common.pr_2094/

File: packages/wix-ui-icons-common/stories/wix-ui-tpa/index.story.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  tab,
+  description,
+  importExample,
+  title,
+  divider,
+} from "wix-storybook-utils/Sections";
+import IconsExample from "../components/icons-example";
+import * as iconComponents from "../../src/wix-ui-tpa/dist";
+import API_Table from "../APITable";
+import WixUiTpaCategoryList from './WixUiTpaCategoryList'
```
Comment (jonathanadler) [inline packages/wix-ui-icons-common/stories/wix-ui-tpa/index.story.tsx:14]: The file name is 
```suggestion
import WixUiTpaCategoryList from './WixTpaCategoryList'
```

So we'll need to either change the file name or the import

**Review by jonathanadler — DISMISSED** [review]
I see the storybook has been updated 🥳

**Review by jonathanadler — APPROVED** [review]
Nice, let's merge!

**Pull request merged**